src/error.rs

```diff
@@ -258,10 +258,12 @@ pub enum Error {
     #[error("the semantic validation of a transaction failed with conflict reason: {} - {0:?}", *.0 as u8)]
     TransactionSemantic(ConflictReason),
     /// Local time doesn't match the time of the latest milestone timestamp
-    #[error("Local time {local_time} doesn't match the time of the latest milestone timestamp: {milestone_timestamp}")]
+    #[error(
+        "Local time {current_time} doesn't match the time of the latest milestone timestamp: {milestone_timestamp}"
+    )]
     TimeNotSynced {
         /// The local time.
```

Comment:
Change the comment as well?

Reply:
But doesn't it make sense like this?

Reply:
Idk, if it makes sense in the comment then it makes sense as a variable name, why is it different?

Reply:
In the error it's about the wrong time, which is your local time, but when we get the time checked, it's also the "current" time from the coordinator
So if we want both the same I would rather say to have all `local_time`, but I think `current_time` sounds better when reading this